commands/src/main/java/org/wildfly/extras/creaper/commands/security/realms/UsernameIsDn.java

```diff
@@ -0,0 +1,43 @@
+package org.wildfly.extras.creaper.commands.security.realms;
+
+/**
+ * Helper class which holds information username-is-dn username to dn in ldap authorization of security realm.
+ */
+public final class UsernameIsDn {
+
+    private final Boolean force;
+    private final LdapCache cache;
+
+    private UsernameIsDn(Builder builder) {
+        this.force = builder.force;
+        this.cache = builder.cache;
+    }
+
+    public Boolean getForce() {
+        return force;
+    }
+
+    public LdapCache getCache() {
+        return cache;
+    }
```

Comment:
Do the methods have to be `public`? I mean, is the class meant to be used for other things than creating a command? If it's only for creating the command, the methods can be omitted and the `private` variables made package-protected.